fetch.bs

```diff
@@ -3916,21 +3912,40 @@ steps:
     happened due to <var>request</var>'s <a>use-CORS-preflight flag</a> being set is
     <a lt="CORS-preflight cache">cached</a>.
 
-   <li><p>If <var>request</var>'s <a for=request>method</a> is not in
-   <var>methods</var>, is not a <a>CORS-safelisted method</a>, and <var>methods</var> does
-   <em>not</em> contain `<code>*</code>`, then return a
-   <a>network error</a>.
+   <li>
+    <p>If all of the following are true
+
+    <ul class=brief>
+     <li><var>request</var>'s <a for=request>method</a> is not in <var>methods</var>
+
+     <li><var>request</var>'s <a for=request>method</a> is not a <a>CORS-safelisted method</a>
+
+     <li><var>request</var>'s <a for=request>credentials mode</a> is not "<code>include</code>"
```

Comment:
The list is "and"-concatenated, so I think splitting this item into two is better.

Reply:
I now think this is wrong. What if credentials mode is "include" and _methods_ does not contain request's method?

Reply:
Thanks, is the following correct?

 - the method is not listed,
 - the method is not CORS-safeliested, and
 - the list doesn't have a wildcard symbol, i.e., not (_credentials_ != "include" and  contains `*`)